activerecord/lib/active_record/suppressor.rb

```diff
@@ -0,0 +1,34 @@
+module ActiveRecord
+  module Suppressor
+    extend ActiveSupport::Concern
+
+    module ClassMethods
+      # Suppresses creation of events for the duration of the block.
```

Comment:
That sentence could also use a tune up. Picture a person who didn't work on this code, what would "creation of events" mean to them? This code really suppresses Active Record save callbacks.